resources/qml/Avatar.qml

```diff
@@ -28,10 +28,17 @@ Rectangle {
         font.pixelSize: avatar.height / 2
         verticalAlignment: Text.AlignVCenter
         horizontalAlignment: Text.AlignHCenter
-        visible: img.status != Image.Ready
+        visible: img.status != Image.Ready && !Settings.useIdenticon
         color: colors.text
     }
 
+    Image {
+        id: identicon
+        anchors.fill: parent
+        visible: img.status != Image.Ready && Settings.useIdenticon
+        source: "image://jdenticon/" + userid
```

Comment:
Don't even load the image, if identicons are disabled.

Reply:
Double review

Reply:
Can this be done by setting the source to null?

Reply:
I'd probably set it to undefined or so.

Reply:
keep in mind, setting source to undefined will result in a QML warning in the logger.  Not sure if that applies if 'visible' is set to false

Reply:
Fixed warning